vmss-prototype/vmss-prototype

```diff
@@ -416,6 +416,10 @@ def image_tweaks(node_name):
             ' /var/lib/cloud/instances/*'
             ' /var/lib/waagent/history/*'
             ' /var/lib/waagent/events/*'
+            # And, clean up the journal logs as they can get
+            # lost on new VMs (old logs just build up)
+            # and journalctl vacuum will not clean them up
+            ' /var/log/journal/*'
```

Comment:
This was an interesting discovery in the latest systems - we had not noticed this before (likely due to other items or prior behaviors) but the systemd journald system will set up a new unique directory name for the journald logs and will never look at the other directories after it does this.  Since it is a new node anyway, it should start with some newer logs so this is actually a good thing but it is required since if we don't do this we can end up building up a whole history abandoned logs which slowly leak disk space over time.  (Or maybe not so slowly, if you up the journald log size limits)